src/components/QuizContainerNew/QuestionsListNew/QuestionListItemNew/QuestionListItem.tsx

```diff
@@ -19,37 +20,39 @@ const QuestionListItem: FC<QuestionListItemProps> = (props) => {
   const { questionStem, answers } = question;
 
   return (
-    <Flex $flexDirection={"column"} $width={"100%"} role="listitem" $gap={8}>
-      <QuestionStem
-        questionStem={questionStem}
-        index={index}
-        showIndex={question.questionType !== "explanatory-text"}
-      />
-
-      {answers && (
-        <>
-          {answers["multiple-choice"] &&
-            answers["multiple-choice"].length > 0 && (
-              <MCAnswers
-                answers={answers["multiple-choice"]}
-                questionNumber={index}
-              />
+    <MathJax hideUntilTypeset="every" dynamic>
```

Comment:
yeah this confused me too but from what I understand the mathjax happens client side but the page is otherwise prerendered, so this flag is required to tell it to rerender when the conversion is complete